### PR TITLE
Wes/sim resources

### PR DESCRIPTION
**Pull request by wesfletch**
Working mapping using the depth cam; fixes to workflow and launch files;

### CODE REVIEW

File: catkin_ws/src/bowser_2dnav/package.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="wes@todo.todo">wes</maintainer>
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>TODO</license>
```
Comment (xar-tol) [inline catkin_ws/src/bowser_2dnav/package.xml:16]: Do?

File: catkin_ws/src/bowser_2dnav/yaml/costmap_common_params.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+footprint: [[0.45,-0.41], [0.45,0.41], [-0.45,0.41], [-0.45,-0.41]]
+footprint_padding: 0.03
+inflation_radius: 0.3
+transform_tolerance: 1
+max_obstacle_height: 9999.9999
+min_obstacle_height: .4
+
+observation_sources: point_cloud_obstacles point_cloud_ground
+
+# point_cloud_sensor1: {
```
Comment (xar-tol) [inline catkin_ws/src/bowser_2dnav/yaml/costmap_common_params.yaml:12]: Can we get rid of this?

File: catkin_ws/src/bowser_sim/models/bowser_hdl32/model.sdf [hunk 3]
```diff
@@ -56,53 +56,51 @@
...
+            <min>0.7</min>
             <max>10</max>
             <resolution>0.02</resolution>
           </range>
         </ray>
         <plugin name="laser" filename="libgazebo_ros_velodyne_laser.so">
             <robotNamespace></robotNamespace>
-            <topicName>/cloud_in</topicName>
-	        	<!-- <topicName>/bowser/velodyne</topicName> -->
+            <!-- <topicName>/velodyne</topicName> -->
```
Comment (xar-tol) [inline catkin_ws/src/bowser_sim/models/bowser_hdl32/model.sdf:102]: can we get rid of this?

**Review by xar-tol — APPROVED** [review]

File: catkin_ws/src/bowser_2dnav/package.xml (hunk 1 above)
Comment (wesfletch) [reply catkin_ws/src/bowser_2dnav/package.xml:16]: No thanks.

File: catkin_ws/src/bowser_2dnav/yaml/costmap_common_params.yaml (hunk 2 above)
Comment (wesfletch) [reply catkin_ws/src/bowser_2dnav/yaml/costmap_common_params.yaml:12]: This is what we call "inline documentation." If it isn't still there, I'll forget about it.

**Review by wesfletch — COMMENTED** [review]
Removed.

**Pull request merged**